include/kansei/mpu.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef KANSEI__MPU_HPP_
+#define KANSEI__MPU_HPP_
+
+#include <string>
+
+#include "keisan/angle.hpp"
+
+#include "./pthread.h"
+
+namespace kansei
+{
+
+class MPU
+{
+public:
+  explicit MPU(const std::string & port_name);
+  ~MPU();
+
+  bool connect();
+
+  void angle_update();
+
+  void reset();
+
+  void set_port_name(const std::string & port_name);
+
+  void set_compensation(keisan::Angle<double> compensation);
+
+  keisan::Angle<double> get_angle();
+  keisan::Angle<double> get_pitch();
+  keisan::Angle<double> get_roll();
+
+  keisan::Angle<double> angle_compensation;
+  keisan::Angle<double> angle_error;
+
+  bool left_;
+  bool right_;
+
+  bool calibrated_;
+
+private:
+  int socket_fd_;
+  std::string serial_name_;
+
+  keisan::EulerAngles rpy;
```

Comment:
I see that ICHIRO's old project was using `trailing_under_score_` naming case for private properties, while the newer project just using `under_score` naming case for all variables and function. But you mixed them all in here, don't you want to fix this inconsistency? It's not critical though, won't affect performance, and in the end the result is same.